### PR TITLE
Add custom padding and border radius to SlidableAction

### PR DESCRIPTION
**Pull request by johnstef99**
# Notes
Custom padding was added because the default padding for the outlined button is kind of large.

# Results
Adding these parameters allow you to create a UI that looks like that:

### CODE REVIEW

**Review by khanguyen-appvity — APPROVED** [review]

File: lib/src/actions.dart [hunk 1]
```diff
@@ -59,13 +61,25 @@ class CustomSlidableAction extends StatelessWidget {
   /// {@endtemplate}
   final bool autoClose;
 
-  /// {@template slidable.actions.onPressed}
-  /// Called when the action is tapped or otherwise activated.
+  /// {@template slidable.actions.onpressed}
+  /// called when the action is tapped or otherwise activated.
   ///
-  /// If this callback is null, then the action will be disabled.
+  /// if this callback is null, then the action will be disabled.
```
Comment (Trulsmatias) [inline lib/src/actions.dart:67]: These capitalization changes looks like mistakes?

Comment (johnstef99) [reply lib/src/actions.dart:67]: yeap 😅 sorry for that

**Comment by johnstef99** [comment]
@letsar what do you think?

**Review by letsar — APPROVED** [review]

**Comment by letsar** [comment]
Thanks @johnstef99 :-)

**Pull request merged**